### PR TITLE
Rename to rrtr-bootstrap

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,11 +84,6 @@
     "yargs": "^4.4.0"
   },
   "files": [
-    "README",
```
Comment (taion) [inline package.json:87]: These are automatically included.

File: package.json [hunk 2]
```diff
@@ -84,11 +84,6 @@
...
-    "CHANGELOG.md",
     "lib"
-  ],
-  "release-script": {
-    "bowerRepo": "git@github.com:react-bootstrap/react-router-bootstrap-bower.git"
```
Comment (taion) [inline package.json:92]: Screw Bower.

**Comment by jquense** [comment]
LGTM

**Comment by taion** [comment]
I'm fixing the Travis config real quick; I'll merge when it's ready.

File: package.json (hunk 2 above)
Comment (jquense) [inline package.json:92]: I never quite understaood the seperate repo? why not just a /dist folder...

Comment (taion) [inline package.json:92]: Most people other than you don't like keeping around built stuff in Git 😛

Comment (jquense) [inline package.json:92]: hehe, not sure why, my way is awesome :P

**Pull request merged**